notebooks/de_dommel/01_fix_model_network.py

```diff
@@ -52,10 +31,7 @@
 # %% verwijder duplicated edges
 
 # see: https://github.com/Deltares/Ribasim-NL/issues/102#issuecomment-2288780504
-model.edge.df = model.edge.df[~((model.edge.df.from_node_type == "Basin") & (model.edge.df.to_node_type == "Basin"))]
```

Comment:
If this can go, should the comment also be removed?

Reply:
Not really, both comments are now solved ad once. Indeed issue #102 is not compatible with the latest version of Ribasim-api, but I don't want to keep issues up-to-date with how-to-fix them in the latest ribasim-version :-)